### ISSUE TITLE
Bug in Laravel Octane

### ISSUE THREAD

**Issue by LikoSoftDev** [outl1ne/nova-multiselect-field#160]
Hi!  In this point http://joxi.ru/8An8ERqcNkl3Om we have the same data between requests and memory leaks when we use Laravel Octane.

**Comment by Tarpsvo** [comment]
Hmm, I'm not even sure how to fix this to be honest. Maybe it's possible to unset the listener somehow?

**Comment by pavloniym** [comment]
Confirmed!
In Octane you should not use such model static methods because of memory leaks. So `multiselect` field will execute this part of code (`model::saved` event)  on each model change (e.g. reorder, or something else which is not related to multiselect relation).
As a simple workaround (without package modification) is to add custom `fillUsing` callback:

```php
Multiselect::make(__('teams/team-user.fields.role'), 'roles')
    ->nullable()
    ->belongsToMany(TeamRoleResource::class, false) // ! order is important
    ->fillUsing(fn (NovaRequest $request, TeamUser $model, string $attribute, string $requestAttribute) => $model->{$attribute}()?->sync($request->get($requestAttribute) ?: []))

```

**Comment by Tarpsvo** [comment]
Heya! Thanks for the report. Should now be fixed in version 4.3.5. ;)

**Comment by pavloniym** [comment]
Cool, thanks!
Waiting for release!
